brambling/mail.py

```diff
@@ -151,10 +154,42 @@ def get_recipients(self):
         return Person.objects.filter(
             Q(owner_orgs=self.order.event.organization) |
             Q(editor_orgs=self.order.event.organization) |
-            Q(editor_events=self.order.event)
+            Q(editor_events=self.order.event),
+            notify_new_purchases=Person.NOTIFY_EACH,
         ).values_list('email', flat=True).distinct()
 
 
+class DailyDigestMailer(FancyMailer):
+    key = "daily_digest"
+
+    def __init__(self, recipient, cutoff, *args, **kwargs):
+        self.recipient = recipient
+        self.cutoff = cutoff
+        from brambling.models import Transaction
+        self.transactions = Transaction.objects.filter(
+            Q(event__additional_editors=recipient) |
+            Q(event__organization__editors=recipient) |
+            Q(event__organization__owner=recipient),
+            timestamp__gte=cutoff,
```

Comment:
As in, we are testing the recipient aspects of this query but not the cutoff aspect.